src/pages/featured-globe/featured-globe-component.jsx

```diff
@@ -41,51 +43,71 @@ const FeaturedGlobeComponent = ({
   clickCallbacksArray,
   mouseMoveCallbacksArray
  }) => {
+  const isMapsList = selectedSidebar === 'featuredMapsList';
+  const [handle, setHandle] = useState(null);
+
+  const spinGlobe = (view) => {
+    loadModules(["esri/core/scheduling"]).then(([scheduling]) => {
+      const spinningGlobe = scheduling.addFrameTask({
+        update: function() {
+          const camera = view.camera.clone();
```

Comment:
Hey @aabdaab this is quite demanding task for the computer (at least mine computer fan is about to take off :). It gets smoother if we move the camera cloning outside the frame task to avoid cloning the camera on every frame.
🙏

Reply:
good point, I'll move it outside! thanks a lot for the review 🙏